Update working_with_projections.rst (#3667)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_projections/working_with_projections.rst

To create a new CRS, click the |signPlus| :sup:`Add new CRS` button and
enter a descriptive name and the CRS parameters.

> Integrate a NTv2-transformation in QGIS 

Reviewer: ```suggestion
Integrate an NTv2-transformation in QGIS 
```

""""""""""""""""""""""""""""""""""""""""""""""

> To integrate a NTv2 transformation file in QGIS you need one more step: 

Reviewer: ```suggestion
To integrate an NTv2 transformation file in QGIS you need one more step: 
```

> 1. Add the NTv2 file (.gsb) in the CRS folder of QGIS (e.g. *C:\\OSGeo4W64\\share\\proj* for windows users)

Reviewer: ```suggestion
1. Place the NTv2 file (.gsb) in the CRS/Proj folder that QGIS uses (e.g. *C:\\OSGeo4W64\\share\\proj* for windows users)
```

> 2. Add in the proj.4 definition the parameter **nadgrids** (+nadgrids=namefile.gsb)

Reviewer: ```suggestion
2. Add **nadgrids** (+nadgrids=nameofthefile.gsb) to the Proj4 definition in the :guilabel:`Parameters` field of the :guilabel:`Custom Coordinate Reference System Dialog` (:menuselection:`Settings --> Custom Projections...`).
```

To create a new CRS, click the |signPlus| :sup:`Add new CRS` button and
enter a descriptive name and the CRS parameters.

Integrate an NTv2-transformation in QGIS 
""""""""""""""""""""""""""""""""""""""""""""""
To integrate an NTv2 transformation file in QGIS you need one more step: 

> 1. Place the NTv2 file (.gsb) in the CRS/Proj folder that QGIS uses (e.g. *C:\\OSGeo4W64\\share\\proj* for windows users)

Reviewer: On Ubuntu (18.04, but also earlier versions) the location is /usr/share/proj/

> 1. Place the NTv2 file (.gsb) in the CRS/Proj folder that QGIS uses (e.g. *C:\\OSGeo4W64\\share\\proj* for windows users)

Author: I confirm the Ubuntu/debian location
